Backport-Guide.md

```diff
@@ -26,77 +99,86 @@ into the `stable-1.2` branch of the `runtime` repo.
     $ git fetch origin
     ```
 
-    And check the list of current stable branches:
+    Check the list of stable branches:
 
     ```bash
     $ cd ${GOPATH}/src/github.com/kata-containers/runtime
     $ git branch -r | grep origin/stable
     ```
 
-- Create your branch to work on:
+- Create your branch to work on, based on the `stable-1.2` branch:
 
     ```bash
     $ git checkout -b my_1.2_pr_backport origin/stable-1.2
     ```
 
-- Locate the commits you want to pull in:
+### Locate commits to cherry pick
 
-    Here we look in the `master` branch, presuming the PR has been merged.
-    If you have the PR in a local branch you can substitute `master` for the name
-    of that branch.
+To list all commits in the `master` branch which are not in the `stable-1.2`
+branch:
 
-    ```bash
-    $ git log --oneline master
-    # And search for the SHA of the commits you wish to backport.
-    ```
+```bash
+$ git log --oneline --no-merges ..master
+```
+
+Make a note of the SHA values for the commits in the PR to backport.
+
+> **Note:** If your PR is in a local branch, substitute `master` for the name
+> of that branch.
+
+### Apply the commits
 
 - Pull in your commits:
 
-    If you are pulling the commits in from the `master` branch, you can add the `-x`
-    argument to `git cherry-pick` to automatically add a reference in the
-    commit to the original commits. This is strongly recommended to aid traceability.
-    If you pick the commits from your local branch do *not* use `-x`. This
-    potentially adds references to SHAs that only exist in your user branches, which
-    is not be useful for future tracability.
+  If you are pulling the commits in from the `master` branch, you can add the `-x`
+  argument to `git cherry-pick` to automatically add a reference in the
+  commit to the original commits. This is strongly recommended to aid traceability.
+  If you pick the commits from your local branch do *not* use `-x`; this
+  potentially adds references to SHAs that only exist in your local branch, which
+  is not useful for future tracability.
 
-    It is also required that you add the `-s` signoff to the commits, if you did not
-    create the original commits.
+  It is also required that you add the `-s` signoff to the commits, if you did not
+  create the original commits.
 
-    ```bash
-    $ git cherry-pick -x <commit>...
-    ```
+  ```bash
+  $ git cherry-pick -x <commit>...
+  ```
+
+  You can cherry pick ranges of commits. Please see the `git-cherry-pick(1)`
+  man page for more information.
 
-    You can cherry pick ranges of commits, etc. Please see the `git-cherry-pick(1)`
-    man page for more information.
+  > *Note:* You do not need to open a new `Issue` for or add an extra `Fixes: nnn` item
```

Comment:
s/for or add/or add/ - maybe - the `for` makes the sentence read funny to me.

Reply:
I think that's the original wording, but agreed - fixed ;)